src/templates.js

```diff
@@ -95,15 +95,14 @@ export const RECIPES_LEARNED = (_, learnedRecipes) => {
   const learnedRecipeNames = learnedRecipes.map(({ name }) => name)
 
   if (learnedRecipes.length === 2) {
-    recipesString = learnedRecipeNames.join(' and ')
+    recipesString = `**${learnedRecipeNames[0]}** and **${learnedRecipes[1]}**`
```

Comment:
Looks like `learnedRecipes` should be `learnedRecipeNames`. 😄

<img width="413" alt="Screen Shot 2021-08-20 at 8 27 00 AM" src="https://user-images.githubusercontent.com/366330/130240681-e2e5a663-9f4c-463e-bff0-1e60656c6fd0.png">

Fixed in https://github.com/jeremyckahn/farmhand/commit/b1566fe4d2dc9cb1710ce7c4cecdb9d00cc99275!

Reply:
good catch!